tests/kafkatest/tests/connect/connect_test.py

```diff
@@ -127,3 +128,73 @@ def validate_output(self, value):
             return output_hash == hashlib.md5(value).hexdigest()
         except RemoteCommandError:
             return False
+
+    @cluster(num_nodes=5)
+    @parametrize(error_tolerance="all")
```

Comment:
these literals for the error tolerance pop up quite a bit in the code, they should probably be constants on ConnectService or somewhere like that.